src/scss/phy/cards.scss

```
@@ -1,62 +1,72 @@
 @use 'sass:math';
 
-.card-neat {
-  .event-card-image {
+// number of news cards per line for each breakpoint
+$news-cards-per-line: (
+  xs: 2,
+  lg: 3
+);
+
+@each $name, $breakpoint in $grid-breakpoints {
+  @if (map-has-key($news-cards-per-line, $name)) {
+    @include media-breakpoint-up($name) {
+      .card-deck .news-card {
+        flex: 0 0 calc(#{math.percentage(math.div(1, map-get($news-cards-per-line, $name)))} - #{$grid-gutter-width});
+        width: 100%;
+      }
+    }
+  }
+}
+
+.pod {
+  background-color: transparent;
+  border: none;
+  width: 318px;
+  flex: none;
+
+  .pod-title {
+    font-size: 16px;
+    font-weight: 700;
+  }
+  
+  .pod-img {
+    border-radius: 8px;
+    height: 158px;
+  }
+
+  .event-pod-img {
     position: relative;
     min-height: 100px;
 
-    .event-card-image-banner {
+    .event-pod-badge {
       position: absolute;
       bottom: 0;
-      background-color: rgba(51,51,51,0.73);
       width: 100%;
-      color: white;
-      text-align: right;
-      padding: 8px 15px 8px;
+      text-align: left;
+      padding: 8px 15px 15px;
 
       &.disabled {
         z-index: 5;
       }
     }
 
-    img {
+    .event-pod-hex {
+      position: absolute;
+      top: 0;
+      right: 5%;
+      min-height: 91px;
       width: 100%;
-      height: 100%;
-    }
-  }
-
-  .event-card-icons {
-    margin-left: auto;
-    img {
-      height: 2rem;
-      width: 2rem;
-      margin-left: 0.25rem;
-    }
-  }
-}
-
-.card-deck .news-card {
-  max-width: 308px;
-  margin-bottom: 1.5rem;
-
-  a img {
-    @include aspect-ratio(5, 3);
-    object-fit: cover;
-  }
-}
+      text-align: right;
+      color: white;
 
-// number of news cards per line for each breakpoint
-$news-cards-per-line: (
-  xs: 2,
-  lg: 3
-);
+      &.disabled {
+        z-index: 5;
+      }
 
-@each $name, $breakpoint in $grid-breakpoints {
-  @if (map-has-key($news-cards-per-line, $name)) {
-    @include media-breakpoint-up($name) {
-      .card-deck .news-card {
-        flex: 0 0 calc(#{math.percentage(math.div(1, map-get($news-cards-per-line, $name)))} - #{$grid-gutter-width});
-        width: 100%;
+      > b {
```

They seem to be using a different font to the body for this. It's embedded in the SVG, but it looks like Exo2 semibold to me.

```suggestion
      > b {
          font-family: $secondary-font-semi-bold;
```